packages/wallets/src/components/OptionsAndMultipliersListing/OptionsAndMultipliersListing.tsx

```diff
@@ -66,9 +67,14 @@ const ShowOpenButton = ({ isExternal, redirect }: TShowButtonProps) => {
 
 const OptionsAndMultipliersListing: React.FC = () => {
     const { isMobile } = useDevice();
+    const { data } = useActiveWalletAccount();
```

Comment:
minor/nitpick: I tend to destructure or at least rename on spot, to avoid non-meangfull variables like "data", 
e.g.
const { data: activeWallet } = useActiveWalletAccount()